pythonFiles/runJediLanguageServer.py

```diff
@@ -5,8 +5,86 @@
 EXTENSION_ROOT = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 sys.path.append(os.path.join(EXTENSION_ROOT, "pythonFiles", "lib", "python"))
 
+import pygls.protocol
+
+
+def is_json_basic_type(obj):
+    """Returns true if the object is an instance of type [int, floor, bool, str]"""
```

Comment:
floor -> float

Reply:
Also, if this is meant to cover Python 2 as well, it should check for `unicode`.